go/vt/vtgate/vindexes/lookup.go

```diff
@@ -181,6 +181,11 @@ func (ln *LookupNonUnique) MarshalJSON() ([]byte, error) {
 	return json.Marshal(ln.lkp)
 }
 
+// IsBackfilling implements the LookupBackfill interface
+func (ln *LookupNonUnique) IsBackfilling() bool {
+	return ln.writeOnly
+}
+
 // Query implements the LookupPlanable interface
 func (ln *LookupNonUnique) Query() (selQuery string, arguments []string) {
 	return ln.lkp.query()
```

Comment:
add this to `ConsistentLookup` as well.

Reply:
Added in 7361fe245816db696c12da9306da125539aacffa, I _think_ using `*clCommon` should take care of that.